src/data/guidelines/color-tokens.js

```diff
@@ -1240,12 +1240,12 @@ const colorTokens = {
           hex: '#8d8d8d',
         },
         g90: {
-          name: 'Gray 50',
-          hex: '#8d8d8d',
+          name: 'Gray 10, 25% opacity',
+          hex: '#f4f4f4 @ 25%',
```

Comment:
Seems like this doesn't match the code. Should `text-on-color` be changed to `Gray10` or should we just use `Gray10` instead of referencing `text-on-color` in `text-on-color-disabled`?

<img width="648" alt="Screen Shot 2022-08-30 at 6 20 14 PM" src="https://user-images.githubusercontent.com/11928039/187489152-91e7ff82-1ee7-40b3-9bef-1fb33a4fdaef.png">


Same issue with the `g100` token

Reply:
Oh oops good catch. That's correct in code. This should be White here.

Reply:
I just say the color because its hard to reference other tokens in this context.